extensions/bland/actions/sendCall/sendCall.ts

```diff
@@ -23,21 +23,52 @@ export const sendCall: Action<
       payload,
     })
 
+    const metaData = {
+      awell_patient_id: payload.patient.id,
+      awell_care_flow_definition_id: payload.pathway.definition_id,
+      awell_care_flow_id: payload.pathway.id,
+      awell_activity_id: payload.activity.id,
+    }
+
     const { data } = await blandSdk.sendCall({
       phone_number: fields.phoneNumber,
       task: fields.task,
+      /**
+       * A POST request will be made to this endpoint when the call ends
+       *
+       * [!!]
+       * Currently only works with the sandbox environment but that's good enough for Workit bootcamp.
+       * Todo: We need to ensure we grab the right base URL, depending on the environment.
+       */
+      webhook: `https://api.sandbox.awellhealth.com/rest/v2/extension_activities/${payload.activity.id}/complete?dp_in_body=call_id`,
```

Comment:
do we want to add any other data points? seems like no?

Reply:
Idd not needed. Any other data we fetch using the "Get call details" action which only needs the call ID.